config.js

```diff
@@ -92,6 +92,20 @@ var config = {
     // Sets the preferred resolution (height) for local video. Defaults to 720.
     // resolution: 720,
 
+    // New style constraints to use for video capture. Currently supported fro
```

Comment:
maybe phrase this as 'used by browsers which return true for `RTCBrowserType.usesNewGumFlow`.

also `fro` -> `for`